content/blog/2019-03-28-incubator-workshop-proposal.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Report from URSSI workshop on research software incubators"
+date: 2019-03-28T10:00:22-07:00
+author: "Nic Weber"
+published: false
+---
+
+## Summary 
+At the first URSSI [community workshop](http://urssi.us/workshops/berkeley/), a small group of participants started to discuss a model for incubating research software projects. Incubation in this context might include a structured program that helps developers plan a new community-based software project, or improve existing projects that need mentorship, strategy, or other resources in order to sustainably grow. 
+
+To further explore this topic we brought together 16 experienced funders, community managers, developers, researchers, and software users. We discussed differences between research software incubators and technology start-up accelerators - identifying important questions about the period of time for incubation, the goals for participants and funders, and potential benefits to a research community. We then heard presentations from 6 different existing programs that have an interest in software development, project management, and incubation. Using these examples we engaged in a design activity to make some initial decisions about an incubator program that URSSI could manage [DSK: IS MANAGE THE RIGHT THING?  AS OPPOSED TO ORGANIZE OR COORDINATE?] in the future.
```

Comment:
Note that I added a note/question here...

Reply:
changed verb to 'organize'